testem.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   test_page: 'tests/index.html?hidepassed',
+  client_decycle_depth: 12,
```

Comment:
Hmm why is this needed?

Reply:
Ask me half a year ago :smile:. I'm afraid I don't remember at this point, but I'd expect it had to do with the default `client_decycle_depth` changing when bumping deps and thus causing test failures.